website_field_autocomplete/demo/field_autocomplete_demo.xml

```diff
@@ -6,31 +6,47 @@
 -->
 
 <odoo>
-    <template id="field_autocomplete_demo" name="Field Autocomplete Demo"  page="True">
-        <t t-call="website.layout">
-            <div id="wrap" class="oe_structure oe_empty">
-                <div class="container">
-                    <h1>Website Menu Autocomplete:</h1>
-                    <div class="row">
-                        <label for="url">URL</label>
-                        <input type="text"
-                               name="url"
-                               class="js_website_autocomplete"
-                               data-query-field="name"
-                               data-display-field="name"
-                               data-value-field="url"
-                               data-limit="10"
-                               data-model="website.menu"
-                               />
+    <record id="field_autocomplete_demo" model="ir.ui.view">
```

Comment:
Just curious... why are you using a `<record>` instead of the previous `<template>`?

Reply:
Because of changes in the `website` module, notably odoo/odoo@4ecbacaf59576a22ff45615a5aa5c67244e4fb93

You can no longer use a `page=True` attribute on the template definition, you have to declare a view & page now (at least that's what I understand of these changes - I took example on [`website` data files](https://github.com/odoo/odoo/blob/ba9381ed2a4765dc5f5f8d18c0e13085ba717815/addons/website/data/website_data.xml#L33-L51))